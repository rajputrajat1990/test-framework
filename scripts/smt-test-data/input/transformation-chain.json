{"user_name": "john", "user_id": "123", "login_time": "1692144000"}
{"user_name": "jane", "user_id": "456", "login_time": "1692144060"}
{"user_name": "bob", "user_id": "789", "login_time": "1692144120"}
