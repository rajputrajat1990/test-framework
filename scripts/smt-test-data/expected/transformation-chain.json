{"full_name": "john", "user_id": 123, "login_time": 1692144000, "processed_at": 1692144000000}
{"full_name": "jane", "user_id": 456, "login_time": 1692144060, "processed_at": 1692144000000}
{"full_name": "bob", "user_id": 789, "login_time": 1692144120, "processed_at": 1692144000000}
